### PR TITLE
P0 card , added homepage , darktheme , updated color scheme , new cards

### PR DESCRIPTION
**Pull request by manojsarna**

### CODE REVIEW

**Review by iRohitGaur — APPROVED** [review]
Nice work! Looks great! Good to merge 👍🏻

**Review by sushmita12345 — APPROVED** [review]
Nice UI. Good to merge👍🏻

**Comment by shruthipatil11** [comment]
Dark mood is great. Ready to merge

File: lib/Components/core.css [hunk 1]
```diff
@@ -93,6 +93,8 @@
          
         /* TEXT COLOR */
         /* DARK THEME */
+
+        --COLOR-OVERLAY:rgb(234, 234, 236,0.6);
         --COLOR-FONT:black;
```
Comment (pankaj-prog) [inline lib/Components/core.css:101]: It doesn't tell which color it is for

**Review by pankaj-prog — APPROVED** [review]
The UI is awesome. Great work Manoj. Good to merge.

File: lib/Components/core.css (hunk 1 above)
Comment (manojsarna) [reply lib/Components/core.css:101]: okay will check

**Pull request merged**